### PR TITLE
Feature/lesson5 homework

### PR DESCRIPTION
**Pull request by aomelchenkopro**

### CODE REVIEW

File: lesson5/Task4.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+if (c > a > b) or (c > b > a):
+    message = f'{c} is the biggest number'
+elif (b > c > a) or (b > a > c):
+    message = f'{b} is the biggest number'
+elif (a > b > c) or (a > c > b):
+    message = f'{a} is the biggest number'
+else:
+    message = "There aren't any average numbers"
+
+print(message)
```
Comment (miha-pavel) [inline lesson5/Task4.py]: последняя СТРОЧКА

File: lesson5/Task5.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# По длинам трех отрезков, введенных пользователем, определить возможность
+# существования треугольника, составленного из этих отрезков.
+# Если такой треугольник существует, то определить, является ли
+# он разносторонним, равнобедренным или равносторонним.
+
+a = int(input('Please, enter the first segment length: '))
+b = int(input('Please, enter the second segment length: '))
+c = int(input('Please, enter the third segment length: '))
+
+fl_existence = False
```
Comment (miha-pavel) [inline lesson5/Task5.py]: НАЗВАНИЕ ПЕРЕМЕННОЙ НЕ СОВСЕМ ИНФОРМАТИВНО

File: lesson5/Task7.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+a = int(input('a = '))
+b = int(input('b = '))
+c = a/b
+d = a % b
+if d == 0:
+    print(f'The first number is divisible by the second. \nThe Result\
+ of the division is {c}')
+else:
+    print(f"The first number isn't divisible by the second without a reminder\
+\nThe result of the division is {c}, {d}")
```
Comment (miha-pavel) [inline lesson5/Task7.py]: последняя линия

File: lesson5/Task4.py (hunk 1 above)
Comment (aomelchenkopro) [reply lesson5/Task4.py]: The mistake was corrected

File: lesson5/Task7.py (hunk 3 above)
Comment (aomelchenkopro) [reply lesson5/Task7.py]: The mistake was corrected

File: lesson5/Task5.py (hunk 2 above)
Comment (aomelchenkopro) [reply lesson5/Task5.py]: The mistake was corrected

**Review by miha-pavel — APPROVED** [review]

**Pull request merged**